functions/that-api-webhooks/.env.sample

```diff
@@ -1,7 +1,14 @@
 SENTRY_NODE_DSN=
 SENTRY_ENVIRONMENT=development
+DEBUG=that:*
+DEBUG_DEPTH=6
+
 # NO SPACES (or tabs)!
 AUTH_CREDS_LIST=<username>:<password>,<username>:<password>
-TALLYFY_ORG_ID=
+
+TALLYFY_ORG_ID=<org id>
```

Comment:
we don't use this..

Reply:
Do you suggest I pull out all the tallyfy code for the docusign to tallyfy webhook?